src/executables/util_cs.c

```diff
@@ -3348,10 +3362,14 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
     check_master_info_end:
       if (error != NO_ERROR)
 	{
-	  PRINT_AND_LOG_ERR_MSG ("%s\n", db_error_string (3));
+	  PRINT_AND_LOG_ERR_MSG ("ERROR : %s\n", db_error_string (3));
+	  check_master_info = false;
+	  error = NO_ERROR;
+	}
+      if (BOOT_IS_CLIENT_RESTARTED ())
+	{
+	  (void) db_shutdown ();
```

Comment:
With this, it seems that you can skip `(void) db_shutdown ();` on 3359L.

Reply:
right, it is duplicated. Thank you.